core/common/src/main/java/alluxio/wire/BlockMasterInfo.java

```diff
@@ -0,0 +1,221 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.wire;
+
+import alluxio.util.FormatUtils;
+
+import com.google.common.base.Objects;
+
+import java.io.Serializable;
+import java.util.HashMap;
+import java.util.Map;
+
+import javax.annotation.concurrent.NotThreadSafe;
+
+/**
+ * The Alluxio block master information.
+ */
+@NotThreadSafe
+public final class BlockMasterInfo implements Serializable {
```

Comment:
I want to change the bytes data into more human-readable information(as what the web UI does).
Another way is to expose the original bytes data in Thrift API and use FormatUtils.getSizeFromBytes() to convert size to string in alluxio-shell alluxio.cli.fsadmin.command.report.SummaryCommand.  
Or maybe our users want original bytes data plus human-readable information (like ....bytes/...GB)?

Reply:
I think it might make sense to keep the thrift API as the raw data, and have the CLI convert it to the readable string. This way, if something else wants the data, it is possible to get the raw data.

Reply:
Agree! I fixed it, thanks